masq_lib/src/utils.rs

```diff
@@ -124,17 +124,30 @@ pub fn exit_process_with_sigterm(message: &str) {
     } else {
         eprintln!("{}", message);
         not_win_cfg::signal::raise(not_win_cfg::signal::SIGTERM).expect("sigterm failure");
-        //This function must not return; so wait for death.
+        //This function must not return; so may it waits for death.
```

Comment:
I think the word "may" doesn't belong here.

Maybe a clearer version: "This function must not return, and the process will be terminated by another thread within micro- or milliseconds, so we wait here for death."